lib/client.js

```diff
@@ -169,20 +175,24 @@ class Client {
    * @example
    *
    * // Using promises
-   * fhirClient.search('Patient', { name: 'smith' }).then((data) => { console.log(data); });
+   * fhirClient.search({ resourceType: 'Patient', identifier: 12345,
```

Comment:
`search` just takes `resourceType` and `searchParams`. Same thing on lines 183-4.